services/order.py

```diff
@@ -0,0 +1,33 @@
+from django.db import transaction
+from db.models import Order, Ticket, User
+from django.db.models.query import QuerySet
+
+
+@transaction.atomic()
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
+) -> None:
+    order = Order.objects.create(
+        user=User.objects.get(
```

Comment:
Use `get_user_model()` instead of `User` here and in other places where applicable as mentioned in the checklist.